tests/utils.ts

```diff
@@ -156,3 +221,65 @@ export async function sendWsMessage(
     };
   });
 }
+
+const MAX_RESPONSE_TIMEOUT = 1_500;
+
+export async function subscriptionRequest(
```

Comment:
Can we add a short summary on the intent/expectation of this method? Especially with the dynamics between status, record and subscription. It seems like it will await forever if records are received, but immediately resolve if subscription/status returned.

Reply:
@thehenrytsai I think I misunderstood your question, are you wondering why resolve at all if it's not subscription and not record?

Because it's resolving a dwn generic response, which could be a failed status for the subscription itself (it wouldn't have a subscription object, or record object).

Reply:
I see, makes sense. All is too say this utility method has a particular use case in mind. Will look at the added summary and make suggestion on the method name if needed.

Reply:
After refactoring the Subscription RPC, some of these util methods were unnecessary. Removed